ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcService.java

```diff
@@ -219,6 +219,7 @@ public CompletableFuture<Void> start() {
 
     final CompletableFuture<Void> resultFuture = new CompletableFuture<>();
     try {
+      NetworkUtility.checkIfPortIsAvailable(config.getPort());
```

Comment:
We also need this check to apply to engineJsonRpc service and probably also the graphQLHttp service.

Reply:
N/A with the new approach I guess